### DISCUSSION TITLE
Checkpoints

### DISCUSSION THREAD

**Discussion by asolano3** [coqui-ai/TTS#1786] [General Q&A]
When I train my model in google colab, I'm not getting any .pth checkpoint files besides the first one.  I need more so I can continue from an interruption (as is common in the timeout fraught colab notebooks). In the config file, I've tried setting "save_checkpoints" to "True" and "save_step" to 15, I've tried "save_all_best" and "save_all_best_after", I've even tried lowering my number of epochs to see if it'll save the checkpoint at the end. But none of this worked.

How do I get checkpoints to save?

This is how I begin my training:

resume_from = "/content/drive/MyDrive/VoiceCloning/output/vits_ljspeech-July-23-2022_02+26AM-0000000/best_model.pth"

!python /content/drive/MyDrive/VoiceCloning/output/vits_ljspeech-July-23-2022_02+26AM-0000000/train_vits.py --restore_path {resume_from}

But I don't have .pth files to continue from besides the first one.

**Comment by domcross** [comment]
You may have a typo in your config. Parameters `save_step` and `save_checkpoints`  in `config.json` should do the job, e.g.
```
"save_step": 10000, // Number of training steps expected to save traning stats and checkpoints.
"save_checkpoints": true, // If true, it saves checkpoints per "save_step"
```
If this does not work for you please share your config...

EDIT: somewhere between v0.5.x and 0.6.x parameter name has changed from `checkpoints` to `save_checkpoints` - which version are you using?
